### DISCUSSION TITLE
How to create comic bubbles with LazPaint

### DISCUSSION THREAD

**Discussion by rchastain** [bgrabitmap/lazpaint#573] [Q&A] [answered]
I would like to create and add comic bubbles to existing pictures, with LazPaint. Would you have some ideas?

**Comment by circular17** [comment, marked as answer]
Sure! Adding comic bubbles to a drawing is a great way to bring it to life. Here is a way to do it with LazPaint.

1. Create the shape of the bubble

- draw an ellipse, set its background to white and its border to thick black. If you loose the focus on the shape, you can go back to it using the _Edit shape_ tool and by clicking on the shape.
- convert the shape to a curve using the _Edit_ menu. This will give you a curve defined by 8 points. It should look something like this:
<img width="387" alt="ellipse-curve" src="https://github.com/bgrabitmap/lazpaint/assets/9092455/6338b987-ac46-49fa-ba28-351027d73978">

- decide where you want the tail of the bubble and add 3 points. To insert a point, hover the mouse to the desired location and press the I key. Like this:
<img width="380" alt="ellipse-three-points" src="https://github.com/bgrabitmap/lazpaint/assets/9092455/a38b8545-a119-4dd1-8ecc-9307977d3df1">

- drag the middle point of the three you just added to form the tail. Then hover the other 2 points and press the `X` letter to ensure they are joined using an angle and not a curve. It should start looking like a bubble now:
<img width="408" alt="bubble-draft" src="https://github.com/bgrabitmap/lazpaint/assets/9092455/af0bc434-b2be-49a2-99c6-6e028e9a313c">

- finally, slightly move away the points around the tail to ensure the roundness of the bubble:
<img width="401" alt="bulle-done" src="https://github.com/bgrabitmap/lazpaint/assets/9092455/f18773e0-38dd-4792-83db-53c3cbb5c639">

2. Add text to your bubble prototype

Add text using the Text tool. Define the rectangular area so that covers the main part of the bubble. Center horizontally and vertically:
<img width="409" alt="bubble-text" src="https://github.com/bgrabitmap/lazpaint/assets/9092455/b1ad4d2d-3cfd-492d-a04d-c87223892f7b">

3. Finalizing the Comic Bubbles:

Once the bubble is ready, you can duplicate the layer for each instance you need. Utilize the move and flip layer options in the layer stack to position them appropriately near the speaking characters.

Your comic bubbles are ready to bring dialogue to your drawings. Different shapes and tail directions can add more dynamism to your scenes. Enjoy!

**Reply by rchastain** [reply]
Perfect tutorial! Thanks.
